src/widgetresize.js

```diff
@@ -29,57 +30,73 @@ export default class WidgetResize extends Plugin {
 	}
 
 	init() {
-		this.resizers = [];
-		this.activeResizer = null;
+		/**
+		 * A map of resizers created using this plugin instance.
+		 *
+		 * @type {Map.<module:engine/view/containerelement~ContainerElement, module:widget/widgetresize/resizer~Resizer>}
+		 */
+		this.resizers = new Map();
+
+		/**
+		 * Currently focused widget resizer instance.
+		 *
+		 * @protected
+		 * @type {module:widget/widgetresize/resizer~Resizer|null}
+		 */
+		this._focusedResizer = null;
 
 		const domDocument = global.window.document;
-		const THROTTLE_THRESHOLD = 16; // 16ms = ~60fps
 
 		this.editor.model.schema.setAttributeProperties( 'width', {
 			isFormatting: true
 		} );
 
 		this._observer = Object.create( DomEmitterMixin );
 
+		// A resizer that is currently used.
+		let activeResizer;
+
 		this._observer.listenTo( domDocument, 'mousedown', ( event, domEventData ) => {
 			if ( !Resizer.isResizeHandle( domEventData.target ) ) {
 				return;
 			}
 
 			const resizeHandle = domEventData.target;
 
-			this.activeResizer = this._getResizerByHandle( resizeHandle );
+			activeResizer = this._getResizerByHandle( resizeHandle );
 
-			if ( this.activeResizer ) {
-				this.activeResizer.begin( resizeHandle );
+			if ( activeResizer ) {
+				activeResizer.begin( resizeHandle );
 			}
 		} );
 
 		this._observer.listenTo( domDocument, 'mousemove', throttle( ( event, domEventData ) => {
-			if ( this.activeResizer ) {
-				this.activeResizer.updateSize( domEventData );
+			if ( activeResizer ) {
+				activeResizer.updateSize( domEventData );
 			}
-		}, THROTTLE_THRESHOLD ) );
+		}, 16 ) ); // 60 fps
```

Comment:
I see absolutely no difference caused by this throttling. If the machine is slow, it does not hit 60fps anyway. With or without throttling, you get the maximum number of fps that it can achieve anyway.

This may only change the behaviour on super fast machines which are able to resize images at more than 60fps, but:

1. it'd have to be 2x faster than mine MBP
2. if you have that fast machine, it doesn't matter if we get more FPS
3. the mousemove event will be throttled by the browser itself because it doesn't try to achieve more than 60fps anyway; see the below timeline with `mousemove` throttled by the browser to exactly 60fps (no manual throttling, I removed entire `updateSize()` call and left a `console.timeStamp()` only):

   ![image](https://user-images.githubusercontent.com/156149/64890452-fca69400-d66f-11e9-8ead-ef9f7a00d4d9.png)

Finally, if anything, we should use `requestAnimationTimeFrame()`, but it simply doesn't seem to be necessary at the moment.

Reply:
I confirmed the same native `mousemove` throttling on Firefox.

Reply:
Throttling is now removed for `mousemove` event.